### DISCUSSION TITLE
Guide to Enterprise Open Source

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#98] [资料分享]
From **Ibrahim Haddad**: _**Hi everyone, I am happy to share my ebook "Guide to Enterprise Open Source" is now available for free download**_
- Link: https://linuxfoundation.org/tools/guide-to-enterprise-open-source/
- [PDF version](https://cta-service-cms2.hubspot.com/ctas/v2/public/cs/c/?cta_guid=6a0bad55-5dd9-4edd-a3f1-a3c48a99f777&signature=AAH58kH6Jk9CMly8hpRpLDRXbPpua2SfLg&placement_guid=72fa2f5f-04d6-47cf-90fb-feaeb1d1a83a&click=25753f0b-475b-4e4e-b388-81bb85f58daf&hsutk=0de571c84464f7e62d8336e500e57dce&canon=https%3A%2F%2Flinuxfoundation.org%2Ftools%2Fguide-to-enterprise-open-source%2F&portal_id=8112310&redirect_url=APefjpEo2z9bh5Hl5BFsDGJYXUQkqsDqpPAnV3Uw6Okgte5DQFhAA1GdcSlNq4RgKu5Pib_KA4SNaiMknS35r8VEhGYt7yOPWwv9xp4xkfmV87Whp7I9JuKjpTRUWYAT-a7ZT6GBHKXEItc1YacJAoQEhBaoVi8rxmcCz24Bg_pRE2WnQD8_nVaqcy8YjmjnNby04UTfxIyrhRO4f4EElAGSTsQReitFSfYnhAMNEi3QU1SLcK8z6mSV2R81g_dUiC3V0K8WsqzwB9_WpeUrVJcOkT9ebGYdWV9DDoVkKdNWG5SDDdpTAQ0mOaBdgYrnhrOI4e83SsE8KYiHK4t4MXPCic1t1_qUORqN3kyrQqNpTFz7QiIMu0X0kZT375FUQHApqQ0YNg6F9MEBnx-4fIF5AO4E7QC7pw&__hstc=160532258.0de571c84464f7e62d8336e500e57dce.1641129731496.1652666953048.1653605969374.24&__hssc=160532258.2.1653605969374&__hsfp=3642067908)

![image](https://user-images.githubusercontent.com/15010826/170593521-4dee8561-d208-4c1a-b38e-d301e44b28e5.png)

Open source software (OSS) has transformed our world and become the backbone of our digital economy and the foundation of our digital world. From the internet and the mobile apps we use daily to the operating systems and programming languages we use to build the future, OSS has played a vital role. It is the lifeblood of the technology industry. 

Organizations involved in building products or services involving software, regardless of their specific industry or sector, are likely to adopt OSS and contribute to open source projects deemed critical to their products and services. Organizations are creating open source program offices (OSPOs) to manage their open source activities, from adopting OSS and compliance with applicable licenses to participating in open standards and foundations. 

To guide their ongoing use of OSS and their engagement with the open source ecosystem, they have developed open source strategies appropriate for their organizational constraints and industry challenges.

The Guide to Enterprise Open Source offers a practical and systematic approach to establishing an open source strategy, developing an implementation plan, and accelerating an organization’s open source efforts.

Authors:
- Dr. **Ibrahim Haddad**, Executive Director of the LF AI & Data Foundation
- With a foreword by David Marr, Vice President, Legal Counsel, Qualcomm Technologies

**Comment by wangleo61** [comment]
写得太全面了，值得读几遍，
刚才看了一遍，里面提到了商业公司参与的开源，很有启发，很多商业公司参与到开源的项目里面，都或多或少和自身的业务和技术背景相关
比如IBM,重金投入Linux,10美金，这个是有长远的战略的考量，中国公司Huawei也是在Linux Kernal贡献度排名很高的。

https://hackernoon.com/commercial-companies-built-on-top-of-an-open-source-world-m0203wcq

Leo

发件人: Will ***@***.***>
发送时间: Friday, May 27, 2022 7:08 AM
收件人: ***@***.***>
抄送: ***@***.***>
主题: [kaiyuanshe/ONES] Guide to Enterprise Open Source (Discussion #98)


From Ibrahim Haddad: Hi everyone, I am happy to share my ebook "Guide to Enterprise Open Source" is now available for free download from https://linuxfoundation.org/tools/guide-to-enterprise-open-source/

  *   Link: https://linuxfoundation.org/tools/guide-to-enterprise-open-source/
  *   PDF version:<https://cta-service-cms2.hubspot.com/ctas/v2/public/cs/c/?cta_guid=6a0bad55-5dd9-4edd-a3f1-a3c48a99f777&signature=AAH58kH6Jk9CMly8hpRpLDRXbPpua2SfLg&placement_guid=72fa2f5f-04d6-47cf-90fb-feaeb1d1a83a&click=25753f0b-475b-4e4e-b388-81bb85f58daf&hsutk=0de571c84464f7e62d8336e500e57dce&canon=https%3A%2F%2Flinuxfoundation.org%2Ftools%2Fguide-to-enterprise-open-source%2F&portal_id=8112310&redirect_url=APefjpEo2z9bh5Hl5BFsDGJYXUQkqsDqpPAnV3Uw6Okgte5DQFhAA1GdcSlNq4RgKu5Pib_KA4SNaiMknS35r8VEhGYt7yOPWwv9xp4xkfmV87Whp7I9JuKjpTRUWYAT-a7ZT6GBHKXEItc1YacJAoQEhBaoVi8rxmcCz24Bg_pRE2WnQD8_nVaqcy8YjmjnNby04UTfxIyrhRO4f4EElAGSTsQReitFSfYnhAMNEi3QU1SLcK8z6mSV2R81g_dUiC3V0K8WsqzwB9_WpeUrVJcOkT9ebGYdWV9DDoVkKdNWG5SDDdpTAQ0mOaBdgYrnhrOI4e83SsE8KYiHK4t4MXPCic1t1_qUORqN3kyrQqNpTFz7QiIMu0X0kZT375FUQHApqQ0YNg6F9MEBnx-4fIF5AO4E7QC7pw&__hstc=160532258.0de571c84464f7e62d8336e500e57dce.1641129731496.1652666953048.1653605969374.24&__hssc=160532258.2.1653605969374&__hsfp=3642067908>

[image]<https://user-images.githubusercontent.com/15010826/170593521-4dee8561-d208-4c1a-b38e-d301e44b28e5.png>

Open source software (OSS) has transformed our world and become the backbone of our digital economy and the foundation of our digital world. From the internet and the mobile apps we use daily to the operating systems and programming languages we use to build the future, OSS has played a vital role. It is the lifeblood of the technology industry.

Organizations involved in building products or services involving software, regardless of their specific industry or sector, are likely to adopt OSS and contribute to open source projects deemed critical to their products and services. Organizations are creating open source program offices (OSPOs) to manage their open source activities, from adopting OSS and compliance with applicable licenses to participating in open standards and foundations.

To guide their ongoing use of OSS and their engagement with the open source ecosystem, they have developed open source strategies appropriate for their organizational constraints and industry challenges.

The Guide to Enterprise Open Source offers a practical and systematic approach to establishing an open source strategy, developing an implementation plan, and accelerating an organization’s open source efforts.

Authors:

  *   Dr. Ibrahim Haddad, Executive Director of the LF AI & Data Foundation
  *   With a foreword by David Marr, Vice President, Legal Counsel, Qualcomm Technologies

―

**Reply by will-ww** [reply]
> 写得太全面了，值得读几遍， 刚才看了一遍，里面提到了商业公司参与的开源，很有启发，很多商业公司参与到开源的项目里面，都或多或少和自身的业务和技术背景相关 比如IBM,重金投入Linux,10美金，这个是有长远的战略的考量，中国公司Huawei也是在Linux Kernal贡献度排名很高的。 

嗯，找个时间我们也分享下~

**Comment by zhuangbiaowei** [comment]
有没有可能申请一下LF的授权，咱们翻译一个中文版？

**Comment by wangleo61** [comment]
好主意呀，算我一个


发件人: ***@***.***>
发送时间: Saturday, May 28, 2022 11:59 AM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] Guide to Enterprise Open Source (Discussion #98)


有没有可能申请一下LF的授权，咱们翻译一个中文版？

―

**Comment by zhuangbiaowei** [comment]
【腾讯文档】A Guide to Enterprise Open Source
https://docs.qq.com/doc/DUmN5dW9nYktDZVFD

目前进展：30%

后续工作：
* 剩余英文内容复制到腾讯文档（预计1小时）
* DeepL初步翻译（预计3小时）
* 中文版PPT排版制作（预计10小时）
* 输出PDF

**Reply by will-ww** [reply]
优秀！

**Reply by tedliu1** [reply]
@zhuangbiaowei @will-ww 
这是我之前整理的企业开源治理相关的介绍，里面引用了不少 Dr. Ibrahim 的前一版内容。
[6-1 企业开源治理.pptx](https://github.com/kaiyuanshe/ONES/files/8803625/6-1.pptx)

**Comment by tedliu1** [comment]
表伟，我之前做过一个 PPT 分享过几次相关内容（应该是 2015 左右的内容，不是这个最新版）。稍晚发出，或许可以参考一下。

Ted Liu 
 
  2022 年 5 月 31 日周二 10:32，Will ***@***.***> 写道：   


优秀！

**Comment by tedliu1** [comment]
表伟，如果需要，我可以分担一些章节哈！

Ted Liu 
 
  2022 年 5 月 31 日周二 ***@***.***> 写道：   


【腾讯文档】A Guide to Enterprise Open Source
https://docs.qq.com/doc/DUmN5dW9nYktDZVFD

目前进展：30%

后续工作：
   
   - 剩余英文内容复制到腾讯文档（预计1小时）
   - DeepL初步翻译（预计3小时）
   - 中文版PPT排版制作（预计10小时）
   - 输出PDF

**Comment by tedliu1** [comment]
表伟，我试试看联系一下 Dr. Ibrahim. 我邀请过他在 COSCon 演讲过，我还有他的微信号。

Ted Liu 
 
  2022 年 5 月 28 日周六 ***@***.***> 写道：   

好主意呀，算我一个


发件人: ***@***.***>
发送时间: Saturday, May 28, 2022 11:59 AM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] Guide to Enterprise Open Source (Discussion #98)


有没有可能申请一下LF的授权，咱们翻译一个中文版？

―

**Comment by wangleo61** [comment]
Ted V5

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: Ted Liu ***@***.***>
发送时间: Tuesday, May 31, 2022 4:53:10 PM
收件人: kaiyuanshe/ONES ***@***.***>
抄送: wangleo61 ***@***.***>; Comment ***@***.***>
主题: Re: [kaiyuanshe/ONES] Guide to Enterprise Open Source (Discussion #98)


@zhuangbiaowei<https://github.com/zhuangbiaowei> @will-ww<https://github.com/will-ww>
这是我之前整理的企业开源治理相关的介绍，里面引用了不少 Dr. Ibrahim 的前一版内容。
6-1 企业开源治理.pptx<https://github.com/kaiyuanshe/ONES/files/8803625/6-1.pptx>

―

**Comment by tedliu1** [comment]
@zhuangbiaowei @wangleo61 
将这个翻译任务加到了 Translation Project 里了。
https://github.com/kaiyuanshe/ONES/projects/3

**Comment by zhuangbiaowei** [comment]
后续工作：

- [x] 剩余英文内容复制到腾讯文档
- [x] DeepL初步翻译（预计3小时）
- [ ] 中文版PPT排版制作（预计10小时）
- [ ] 输出PDF

**Comment by wangleo61** [comment]
表哥太给了

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: ***@***.***>
发送时间: Tuesday, May 31, 2022 5:46 PM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] Guide to Enterprise Open Source (Discussion #98)


后续工作：

  *   [X] 剩余英文内容复制到腾讯文档
  *   [ ] DeepL初步翻译（预计3小时）
  *   [ ] 中文版PPT排版制作（预计10小时）
  *   [ ] 输出PDF

―

**Reply by tedliu1** [reply]
@zhuangbiaowei @will-ww 
1. 翻译基本完成，还剩下图片里的文字尚未翻译。
2. Linux 基金会已经开始安排翻译，以透过邮件和作者 Dr. Ibrahim 沟通，他请我们和 LF APAC 的 Hali Tsang 女士联系。Hali 很欢迎开源社参与翻译的协同贡献。

【腾讯文档】A Guide to Enterprise Open Source
https://docs.qq.com/doc/DUmN5dW9nYktDZVFD

**Reply by will-ww** [reply]
太厉害了！

**Comment by wangleo61** [comment]
WoW, thanks Will, Great sharing, It’s valuable read it.

Leo
发件人: Will ***@***.***>
发送时间: Friday, May 27, 2022 7:08 AM
收件人: ***@***.***>
抄送: ***@***.***>
主题: [kaiyuanshe/ONES] Guide to Enterprise Open Source (Discussion #98)


From Ibrahim Haddad: Hi everyone, I am happy to share my ebook "Guide to Enterprise Open Source" is now available for free download from https://linuxfoundation.org/tools/guide-to-enterprise-open-source/

  *   Link: https://linuxfoundation.org/tools/guide-to-enterprise-open-source/
  *   PDF version:<https://cta-service-cms2.hubspot.com/ctas/v2/public/cs/c/?cta_guid=6a0bad55-5dd9-4edd-a3f1-a3c48a99f777&signature=AAH58kH6Jk9CMly8hpRpLDRXbPpua2SfLg&placement_guid=72fa2f5f-04d6-47cf-90fb-feaeb1d1a83a&click=25753f0b-475b-4e4e-b388-81bb85f58daf&hsutk=0de571c84464f7e62d8336e500e57dce&canon=https%3A%2F%2Flinuxfoundation.org%2Ftools%2Fguide-to-enterprise-open-source%2F&portal_id=8112310&redirect_url=APefjpEo2z9bh5Hl5BFsDGJYXUQkqsDqpPAnV3Uw6Okgte5DQFhAA1GdcSlNq4RgKu5Pib_KA4SNaiMknS35r8VEhGYt7yOPWwv9xp4xkfmV87Whp7I9JuKjpTRUWYAT-a7ZT6GBHKXEItc1YacJAoQEhBaoVi8rxmcCz24Bg_pRE2WnQD8_nVaqcy8YjmjnNby04UTfxIyrhRO4f4EElAGSTsQReitFSfYnhAMNEi3QU1SLcK8z6mSV2R81g_dUiC3V0K8WsqzwB9_WpeUrVJcOkT9ebGYdWV9DDoVkKdNWG5SDDdpTAQ0mOaBdgYrnhrOI4e83SsE8KYiHK4t4MXPCic1t1_qUORqN3kyrQqNpTFz7QiIMu0X0kZT375FUQHApqQ0YNg6F9MEBnx-4fIF5AO4E7QC7pw&__hstc=160532258.0de571c84464f7e62d8336e500e57dce.1641129731496.1652666953048.1653605969374.24&__hssc=160532258.2.1653605969374&__hsfp=3642067908>

[image]<https://user-images.githubusercontent.com/15010826/170593521-4dee8561-d208-4c1a-b38e-d301e44b28e5.png>

Open source software (OSS) has transformed our world and become the backbone of our digital economy and the foundation of our digital world. From the internet and the mobile apps we use daily to the operating systems and programming languages we use to build the future, OSS has played a vital role. It is the lifeblood of the technology industry.

Organizations involved in building products or services involving software, regardless of their specific industry or sector, are likely to adopt OSS and contribute to open source projects deemed critical to their products and services. Organizations are creating open source program offices (OSPOs) to manage their open source activities, from adopting OSS and compliance with applicable licenses to participating in open standards and foundations.

To guide their ongoing use of OSS and their engagement with the open source ecosystem, they have developed open source strategies appropriate for their organizational constraints and industry challenges.

The Guide to Enterprise Open Source offers a practical and systematic approach to establishing an open source strategy, developing an implementation plan, and accelerating an organization’s open source efforts.

Authors:

  *   Dr. Ibrahim Haddad, Executive Director of the LF AI & Data Foundation
  *   With a foreword by David Marr, Vice President, Legal Counsel, Qualcomm Technologies

―

**Comment by wangleo61** [comment]
太给力了，表哥，我可以分担一下任务哈


发件人: Ted ***@***.***>
发送时间: Tuesday, May 31, 2022 11:15 AM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] Guide to Enterprise Open Source (Discussion #98)

表伟，我试试看联系一下 Dr. Ibrahim. 我邀请过他在 COSCon 演讲过，我还有他的微信号。

Ted Liu

2022 年 5 月 28 日周六 ***@***.***> 写道：

好主意呀，算我一个


发件人: ***@***.***>
发送时间: Saturday, May 28, 2022 11:59 AM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] Guide to Enterprise Open Source (Discussion #98)


有没有可能申请一下LF的授权，咱们翻译一个中文版？

D
Reply to this email directly, view it on GitHub<https://github.com/kaiyuanshe/ONES/discussions/98#discussioncomment-2839096>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AO5RA3YJXEHVUI55Q3PTUCDVMGKYRANCNFSM5XCTHD2A>.
You are receiving this because you commented.Message ID: ***@***.***>


―
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>



―

**Comment by wangleo61** [comment]
太给力了

发件人: Ted ***@***.***>
发送时间: Wednesday, June 1, 2022 12:54 AM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] Guide to Enterprise Open Source (Discussion #98)


@zhuangbiaowei<https://github.com/zhuangbiaowei> @will-ww<https://github.com/will-ww>

  1.  翻译基本完成，还剩下图片里的文字尚未翻译。
  2.  Linux 基金会已经开始安排翻译，以透过邮件和作者 Dr. Ibrahim 沟通，他请我们和 LF APAC 的 Hali Tsang 女士联系。Hali 很欢迎开源社参与翻译的协同贡献。

【腾讯文档】A Guide to Enterprise Open Source
https://docs.qq.com/doc/DUmN5dW9nYktDZVFD

―
